### DISCUSSION TITLE
How to set the minimum gwei threshold on PoS

### DISCUSSION THREAD

**Discussion by ghost** [0xPolygon/polygon-edge#915] [Q&A]
Good afternoon. I'm trying to set the minimum gwei to 7150000, so that no transaction goes through too cheap in the quiet times of day. At the moment, I can put anything I like in the gwei box, and it will say it's too low, but still completes the transaction without an issue. What do I need to edit to get the limit set. I have --price-limit 7150000 set in the seal line.

Thanks,
Nex

**Comment by laviniat1996** [comment]
Hi @Nexsuperne101 ,

Did you set the price limit to every node? It needs to be set across all nodes.

**Comment by ghost** [comment]
Thank you for your reply. I have set it on all 10 bootnodes, but not in the
seal line of the validator nodes.

Thanks
Nex

**Reply by laviniat1996** [reply]
@Nexsuperne101 I don't fully understand. What do you mean by seal line of validator nodes?

**Comment by ghost** [comment]
I pasted this into each bootnode (obviously changed the data-dir and IP
address). They all say 7150000 for the --price-limit.

polygon-edge server --data-dir ./node5 --chain genesis.json  --libp2p
0.0.0.0:1478 --jsonrpc 0.0.0.0:10002 --price-limit 7150000 --nat
18.133.180.183
--seal

For the non-bootnodes, it says this;
polygon-edge server --data-dir ./node99 --chain genesis.json --libp2p
0.0.0.0:1478 --jsonrpc 0.0.0.0:10002 --seal

Thanks,
Nex

**Reply by ghost** [reply]
polygon-edge server --data-dir ./node99 --chain genesis.json --libp2p 0.0.0.0:1478 --jsonrpc 0.0.0.0:10002 --price-limit 7150000 --seal
 Should the non-bootnode say this then?
